File: 1semana/aula3/Pessoa.cpp
#include <string>
#include <iostream>
using namespace std;

class Pessoa{

private:
    string nome;
    int idade;

public:

    Pessoa(string n, int i): nome(n), idade(i){}
    void setNome(string n){
        nome = n;
    }
    void setIdade(int i){
        idade = i;
    }
    string getNome(){
        return nome;
    }
    int getIdade(){
        return idade;
    }

};

int main(){

    Pessoa p("Antonio", 35);

    cout << "Nome: "<< p.getNome()<< endl;
    cout << "idade: "<< p.getIdade()<< endl;

    p.setNome("Josevaldo");
    p.setIdade(25);

    cout << "Nome: "<< p.getNome()<< endl;
    cout << "idade: "<< p.getIdade()<< endl;
    
    // o trecho a baixo não funciona pois os dados são privados
    //cout << "Nome: "<< p.nome()<< endl;
    //cout << "idade: "<< p.idade()<< endl;

    return 0;
}

File: 1semana/aula3/Pessoa.java
public class Pessoa{
    private String nome;
    private int idade;

    // construtor
    public Pessoa(String nome, int idade){
        this.nome = nome;
        this.idade = idade;
    }
    public void setNome(String nome){
        this.nome = nome;
    }
    public String getNome(){
        return this.nome;
    }
    public void setIdade(int idade){
        this.idade = idade;
    }
    public int getIdade(){
        return this.idade;
    }
    public static void main(String[] args){

        Pessoa p = new Pessoa("Josevaldo", 25);

        System.out.println("\nNome: " + p.getNome());
        System.out.println("Idade: "+ p.getIdade());


        p.setNome("Antonio");
        p.setIdade(27);

        System.out.println("\nNome: " + p.getNome());
        System.out.println("Idade: "+ p.getIdade());
    }
}